Makefile

```diff
@@ -4,29 +4,37 @@
 IMAGES=echosrv # plus the combo image / Dockerfile without ext.
 
 DOCKER_PREFIX := docker.io/istio/fortio
+LINTERS_IMAGE := docker.io/fortio/fortio.build:v4
 
 TAG:=$(USER)$(shell date +%y%m%d_%H%M%S)
 
 DOCKER_TAG = $(DOCKER_PREFIX)$(IMAGE):$(TAG)
 
 # Local targets:
-install: test
+install:
 	go install ./...
 
+# Local test
 test:
-	go test -timeout 45s -race ./...
+	go test -timeout 60s -race ./...
 
-# Run once
-install-linters:
-	go get -u github.com/alecthomas/gometalinter
-	gometalinter -i -u
+# To debug linters, uncomment
+#DEBUG_LINTERS="--debug"
+
+local-lint:
+	gometalinter $(DEBUG_LINTERS) \
+	--deadline=180s --enable-all --aggregate \
+	--exclude=.pb.go --disable=gocyclo --line-length=132 $(LINT_PACKAGES)
```

Comment:
curious: is this `line-length` consistent with istio/istio ?

Reply:
it's shorter (I think mixer was 160 when I looked)

Reply:
i wonder if we should use a standard length across the effort (silly, i know, but might make it easier to hop repos).